src/main.rs

```diff
@@ -142,12 +163,33 @@ fn compute_verdict(output: &Path, expected_output: &Path) -> Result<Verdict> {
     output.read_to_string(&mut output_str)?;
     expected_output.read_to_string(&mut ex_output_str)?;
     if output_str.trim() != ex_output_str.trim() {
```

Comment:
Will that mean solutions with an additional line in the end would fail ?

Reply:
Why would they? (we still compare the trimmed version of both so it should be fine)

Reply:
Just double checking, as some submissions (most?) are adding a new line at the end